doc/lua_api.txt

```diff
@@ -5791,24 +5875,24 @@ Used by `minetest.register_craft`.
         recipe = {
             {'default:cobble', 'default:cobble', 'default:cobble'},
             {'', 'default:stick', ''},
-            {'', 'default:stick', ''}, -- Also groups; e.g. 'group:crumbly'
+            {'', 'default:stick', ''},  -- Also groups; e.g. 'group:crumbly'
         },
-        replacements = --[[<optional list of item pairs,
-                        replace one input item with another item on crafting>]]
+        -- Optional; replace one input item with another item on crafting
```

Comment:
Should be below 'replacements', maybe worded like this:
"-- 'replacements' is optional: replace ..."

Reply:
or
```
`replacements`: Optional; replace...
```